sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/v2/DescribeTableExec.scala

```diff
@@ -31,6 +31,9 @@ case class DescribeTableExec(
     table: Table,
     isExtended: Boolean) extends V2CommandExec {
 
+
+  override def producedAttributes: AttributeSet = outputSet
```

Comment:
Shall we do this in `V2CommandExec`?

Reply:
Yea, I agree. I was wondering if it is relevant to this PR.